js/gate.js

```diff
@@ -1979,8 +1979,14 @@ module.exports = class gate extends Exchange {
         const bid = this.safeString (ticker, 'highest_bid');
         const high = this.safeString (ticker, 'high_24h');
         const low = this.safeString (ticker, 'low_24h');
-        const baseVolume = this.safeString2 (ticker, 'base_volume', 'volume_24h_base');
-        const quoteVolume = this.safeString2 (ticker, 'quote_volume', 'volume_24h_quote');
+        let baseVolume = this.safeString2 (ticker, 'base_volume', 'volume_24h_base');
+        if (baseVolume === 'nan') {
+            baseVolume = '0';
```

Comment:
Does the 'nan' mean literally "zero volume" or undefined volume?

Reply:
I only see `nan` in future, eg:
```
'BTC/USD:BTC-220819': {
    symbol: 'BTC/USD:BTC-220819',
    timestamp: undefined,
    datetime: undefined,
    high: 0,
    low: 0,
    bid: undefined,
    bidVolume: undefined,
    ask: undefined,
    askVolume: undefined,
    vwap: undefined,
    open: undefined,
    close: 0,
    last: 0,
    previousClose: undefined,
    change: undefined,
    percentage: 0,
    average: undefined,
    baseVolume: 0,
    quoteVolume: 0,
    info: {
      basis_value: '-22.84',
      total_size: '0',
      basis_rate: '-0.030043',
      volume_24h_quote: '0',
      volume_24h_settle: 'nan',
      high_24h: '0',
      change_percentage: '0',
      mark_price: '23100.97',
      index_price: '23123.81',
      volume_24h: '0',
      volume_24h_usd: '0',
      last: '0',
      settle_price: '0',
      volume_24h_btc: '0',
      contract: 'BTC_USD_20220819',
      low_24h: '0',
      volume_24h_base: 'nan'
    }
  },
  'BTC/USD:BTC-220812': {
    symbol: 'BTC/USD:BTC-220812',
    timestamp: undefined,
    datetime: undefined,
    high: 0,
    low: 0,
    bid: undefined,
    bidVolume: undefined,
    ask: undefined,
    askVolume: undefined,
    vwap: undefined,
    open: undefined,
    close: 0,
    last: 0,
    previousClose: undefined,
    change: undefined,
    percentage: 0,
    average: undefined,
    baseVolume: 0,
    quoteVolume: 0,
    info: {
      basis_value: '-33.99',
      total_size: '0',
      basis_rate: '-0.107301',
      volume_24h_quote: '0',
      volume_24h_settle: 'nan',
      high_24h: '0',
      change_percentage: '0',
      mark_price: '23089.82',
      index_price: '23123.81',
      volume_24h: '0',
      volume_24h_usd: '0',
      last: '0',
      settle_price: '0',
      volume_24h_btc: '0',
      contract: 'BTC_USD_20220812',
      low_24h: '0',
      volume_24h_base: 'nan'
    }
  },
```
The btc 24h volume is 0, so I use 0. Should reface with `undefined`?

Reply:
@sc0Vu yep it makes sense I guess, tho this is probably a bug on their end. I will ask them and if needed we revisit this logic, going to merge it so that we can unblock the user